fares-data-build-tool/create-data
# feat: CFD-267 Point to point period netex

# Description

-   Point to point period netex

# Testing instructions

-   Go through the journey for a P2P Period fare and check the Netex at the end
-   Go through for 2 random other journeys and check the Netex generates as expected

# Type of change
